services/hanke-service/src/integrationTest/kotlin/fi/hel/haitaton/hanke/hakemus/HakemusServiceITest.kt

```diff
@@ -735,9 +740,8 @@ class HakemusServiceITest(
                 hakemusFactory.builder(hankeEntity = hanke).withMandatoryFields().saveEntity()
             val applicationData = application.applicationData as CableReportApplicationData
             val expectedDataAfterSend = applicationData.copy(pendingOnClient = false)
-            every { cableReportService.create(any()) } returns alluId
-            every { cableReportService.getApplicationInformation(alluId) } throws
-                AlluException(listOf())
+            every { alluClient.create(any()) } returns alluId
+            every { alluClient.getApplicationInformation(alluId) } throws AlluException(listOf())
```

Comment:
Is the list of errors given for AlluException used?

Reply:
I wondered why this sounded familiar, but it is refactored in https://github.com/City-of-Helsinki/haitaton-backend/pull/667/

Reply:
Yes, I didn't do any changes here because of that. This will need to be fixed if that PR is merged before this one.